apps/css-workshop/src/components/Checkbox.astro

```diff
@@ -0,0 +1,71 @@
+---
+type Props = {
+  isLoading?: boolean;
+  indeterminate?: boolean;
+  checked?: boolean;
+  disabled?: boolean;
+  variant?: 'eyeball';
+  basic?: boolean;
+  status?: 'negative' | 'positive' | 'warning';
+} & astroHTML.JSX.HTMLAttributes;
```

Comment:
I think some of these props don't need to be re-specified since they are part of the input itself.

`checked` and `disabled` can come from `astroHTML.JSX.InputHTMLAttributes`

Reply:
resolved this on a new branch. pr soon